clang/test/CIR/CodeGen/pass-object-size.c

```diff
@@ -0,0 +1,30 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s -check-prefix=CIR
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-llvm %s -o %t.ll
+// RUN: FileCheck --input-file=%t.ll %s -check-prefix=LLVM
+void b(void *__attribute__((pass_object_size(0))));
+void e(void *__attribute__((pass_object_size(2))));
+void c() {
+  int a;
+  int d[a];
+  b(d);
+  e(d);
+}
+
+// CIR: cir.func no_proto @c() extra(#fn_attr)
```

Comment:
This one could be slightly better `CIR: cir.func no_proto @c()`, since we are not testing anything that involves `extra`.